src/Draggable/Sensors/DragSensor/tests/DragSensor.test.js

```diff
@@ -0,0 +1,514 @@
+import {
+  createSandbox,
+  triggerEvent,
+  listenToSensorEvents,
+  restoreSensorEvents,
+  getSensorEventsByType,
+} from 'helper';
+
+import DragSensor from './..';
+
+const sampleMarkup = `
+  <ul>
+    <li>First item</li>
+    <li>Second item</li>
+  </ul>
+`;
+
+/**
+ * Jest context is unaware of the `DataTransfer` constructor normally found in
+ * the browser, so we must rely on a stub to assert on these properties of
+ * `DragEvent.dataTransfer`.
+ *
+ * I'm not sure why this is the case, but it seems to be an issue for other
+ * drag and drop libraries and testing contexts as well. See below.
+ *
+ * @todo Find a way to test this behavior using the real objects
+ * @see https://reactjsnews.com/testing-drag-and-drop-components-in-react-js
+ * @see https://github.com/angular-dragdrop/angular-dragdrop/issues/22
+ *
+ * @return {Object}
+ */
```

Comment:
I think it's legit to mock the `DataTransfer` instance here, no need to go into detail or look for an alternative solution IMO. I'd just leave:

```js
/**
 * Manually firing drag events will require us to mock the `DragEvent.dataTransfer`
 *
 * @return {Object}
 */
```